.golangci.yml

```diff
@@ -23,6 +23,14 @@ issues:
         - gochecknoglobals
         - lll
         - gomnd
+    # tags.go and users.go are called by dupl in entirety. Nolints on package level are not recommended,
```

Comment:
I wasn't aware that those files are duplicated as a whole. In that case whole file could be excluded.  I checked in docs that there are no package-level excludes. So the simplest solution is to put nolint statement in a file (but maybe not above `package` statement so that it would not be a package docstring, if possible).

https://golangci-lint.run/usage/false-positives/

Reply:
It seems //nolint directive is [ignored by godoc](https://github.com/golangci/golangci-lint/issues/892).
I will revert the changes to the previous commit as there is also no other way to nolint an entire file in another place.

Reply:
Done